### DISCUSSION TITLE
Can I group all updates into a single PR per-manifest, but with some exceptions?

### DISCUSSION THREAD

**Discussion by allanlewis** [renovatebot/renovate#24147] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

One PR per manifest with all updates, with some specific exceptions.

### What you tried so far.

Package rules - I can't work out the correct config.

### Relevant debug logs

N/A - I'm not sure where to start with this.

**Comment by rarkins** [comment, marked as answer]
Yes, you'd usually use `matchPackagePatterns` with `*` to match all, and then `excludePackageNames` to exclude whatever you want, and then set a `groupName`.
